src/main/kotlin/racing/model/Car.kt

```diff
@@ -1,17 +1,21 @@
 package racing.model
 
-data class Car constructor(private val position: Int) {
+data class Car constructor(private val position: Int, private val name: String = "unknown") : Comparable<Car> {
```

Comment:
constructor는 생략하셔도 됩니다 😄

Reply:
Comparable 구현 👍

Reply:
```suggestion
data class Car(
    private val position: Int,
    private val name: String = "unknown"
) : Comparable<Car> {
```
두 개 이상의 프로퍼티를 캡슐화하는 경우에는 개행을 해주면 좀 더 보기 좋아져요

Reply:
헛...😓 죄송합니닷...!!